lib/hanami/generators/application/container/README.md.tt

```diff
@@ -1,17 +1,31 @@
 # <%= config[:project_name] %>
 
-Welcome to your new hanami project!
+Welcome to your new Hanami project!
 
-TODO: Delete this and the text above, and describe your project
+## Setup
 
-## Installation
+How to run tests:
+```
```

Comment:
@davydovanton Please leave a blank line between the paragraph and the beginning of the three backticks. Some Markdown renderer are getting confused by this. For instance iA Writer on Mac.